lua/wikis/commons/Infobox/Person.lua

```diff
@@ -353,7 +412,18 @@ end
 ---@param args table
 ---@return {store: string, category: string}
 function Person:getPersonType(args)
-	return {store = 'Player', category = 'Player'}
+	local playerValue = {store = 'player', category = 'Player'}
+	local staffValue = {store = 'staff', category = 'Staff'}
+	local inGameRoles = Roles.InGameRoles
+
+	local isStaff = Array.all(self.roles, function(roleData)
```

Comment:
```suggestion
	local isStaff = Logic.isNotEmpty(self.roles) and Array.all(self.roles, function(roleData)
```
else pages without set roles will be made staff, while currently they are players